packages/odyssey-react/src/components/Icon/Icon.tsx

```diff
@@ -11,49 +11,46 @@
  */
 
 import {
-  Children,
-  cloneElement,
   ComponentPropsWithRef,
   forwardRef,
+  ForwardRefExoticComponent,
 } from "react";
-import type { ReactElement } from "react";
-import { useOid, useOmit } from "../../utils";
+import { useOmit } from "../../utils";
+import type { IconNames } from "./";
+import { iconNameToClassName } from "./";
+import * as Icons from "./";
```

Comment:
This will import every Icon into the bundle whenever our `Icon` component is used here. That seems less than ideal - what are we trying to accomplish here?